code/src/components/HeartCounter.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const HeartCounter = ({ heartCount, setHeartCounter }) => {
+  const removeHeartCounter = () => {
+    setHeartCounter('hidden');
```

Comment:
I've noticed you only have to possible versions of this property - `heartCounter  = 'hidden'` or `heartCounter = 'visible'`. In this kind of situations we should rather use boolean values :

```
const [heartCounterVisibility, setHeartCounterVisibility] = useState(false);
```